contracts/external/traders/Expiry.sol

```diff
@@ -56,23 +53,41 @@ contract Expiry is
 
     bytes32 constant FILE = "Expiry";
 
+    uint256 constant FOUR_WEEKS_IN_SECONDS = 60 * 60 * 24 * 28;
```

Comment:
I think we should make this passed in and not static. We can have some minimum we can set it to